recipes/CheckmarxPythonSDK/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "CheckmarxPythonSDK" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1793313d6b36e538289e587d3fde6a856d07551a5eb61ce291b09292a5a9e898
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - urllib3 ==1.26.9
```

Package metadata says 1.26.8

```suggestion
    - urllib3 ==1.26.8
```

---

Done